include/simdjson/padded_string-inl.h

```diff
@@ -185,6 +191,57 @@ inline simdjson_result<padded_string> padded_string::load(std::string_view filen
   return s;
 }
 
+#if defined(_WIN32) && __cplusplus >= 201703L
```

Comment:
I don't think we can assume that `__cplusplus` will be defined under a Microsoft platform use `SIMDJSON_CPLUSPLUS17`.